### ISSUE TITLE
Adding a geometry to geometry less record with the "add part" tool is cumbersome

### ISSUE THREAD

**Issue by gioman** [qgis/QGIS#41298]
It would be more straightforward if a geometry could be attached (to geometry less record) with the "add polygon/line/point" tool.

**Comment by roya0045** [comment]
The existing tool to create new features (i,e: Add Polygon Feature )could be renamed to 'Create New Feature' as the 'add part' chould simply be rename 'add Feature'. This would show that one add a new object and the other append to an existing object.

**Comment by uclaros** [comment]
A *feature* is an entry in the attribute table and is not necessarily tied to a geometry, so using the *Add polygon feature* to add a geometry to an existing feature would be more strange.
Technically the current implementation is correct: on an existing selected feature, use add part to add a geometry part. When existing geometry has zero parts, the tool will add one part so a singlepart geometry is born. When the geometry already has one or more parts then a multipart geometry is created.

**Comment by roya0045** [comment]
Yeah I mixed up my terms, though I still think renaming "Add Part" to "Add Geometry" would clear things up and promote that usecase.

**Comment by gioman** [comment]
> A _feature_ is an entry in the attribute table and is not necessarily tied to a geometry, so using the _Add polygon feature_ to add a geometry to an existing feature would be more strange.
> Technically the current implementation is correct: on an existing selected feature, use add part to add a geometry part. When existing geometry has zero parts, the tool will add one part so a singlepart geometry is born. When the geometry already has one or more parts then a multipart geometry is created.

@uclaros I understand what you say, but semantic aside it is still very weird for common user to have to use the "add part" tool in this scenario.
